exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `method` (optional): HTTP method, one of: `POST` or `PUT`. `POST` is used by default if unset.
+- `compress` (optional): Option to enable compression (default false)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
```

Comment:
1mb before compression

Reply:
`By default 1MB is recommended.` ?

Reply:
Sorry was not clear. What I meant was I believe the collection teams 1mb batches pre-compression. That was all I was calling out. Was not sure if this 1mb is ore compression or post compression.

Reply:
As I understand this description says 1mb pre-compression